src/theme/components/link.ts

```diff
@@ -0,0 +1,42 @@
+const variants = {
```

Comment:
Where are the `focus` styles coming from?  The default focus blue glow applied by Chakra needs to be removed from the styling.
```box-shadow: 0;```

Reply:
I'll update the overall NYPL theme.